test/appium/tests/atomic/chats/test_activity_center.py

```diff
@@ -0,0 +1,251 @@
+        pub_2.send_message_button.click()
+
+        if not self.home_1.notifications_unread_badge.is_element_displayed():
+            self.errors.append("Unread badge is NOT shown after receiving mentions from Group and Community")
+        self.home_1.notifications_unread_badge.wait_and_click(30)
+
+        self.home_1.just_fyi("Check there are two notifications from two chats are present in Activity Center")
+        if self.home_1.get_chat_from_activity_center_view(self.username_2).chat_message_preview == group_chat_message:
+            self.home_1.just_fyi("Open group chat where user mentioned and return to Activity Center")
+            self.home_1.get_chat_from_activity_center_view(self.username_2).click()
+            self.home_1.home_button.double_click()
+            self.home_1.notifications_button.click()
+        else:
+            self.errors.append("No mention in Activity Center for Group Chat")
+
+        if self.home_1.get_chat_from_activity_center_view(self.username_2).chat_message_preview == channel_message:
+            self.home_1.just_fyi("Open community chat where user mentioned and return to Activity Center")
+            self.home_1.get_chat_from_activity_center_view(self.username_2).click()
+            self.home_1.home_button.double_click()
+        else:
+            self.errors.append("No mention in Activity Center for community chat")
+
+        self.home_1.just_fyi("Check there are no unread messages counters on chats after message read")
+        if (self.home_1.notifications_unread_badge.is_element_present() or
+                self.home_1.get_chat_from_home_view(self.group_chat_name_3).new_messages_counter.text == "1" or
+                self.home_1.get_chat_from_home_view(community_name).new_messages_counter.text == "1"):
+            self.errors.append("Unread message indicator is kept after all messages read in chats")
+
+        self.home_1.just_fyi("Check there is an empty view on Activity Center")
+        self.home_1.notifications_button.click()
+        if not self.home_1.element_by_translation_id('empty-activity-center').is_element_present():
+            self.errors.append("It appears Activity Center still has some chats after user opened all of them")
+
+        self.device_1.just_fyi('Device1 removes Device2 from contacts (for the next test)')
+        self.home_1.home_button.double_click()
+        self.home_1.get_chat_from_home_view(self.username_2).click()
+        self.device_1_one_to_one_chat = self.home_1.get_chat_view()
+        self.device_1_one_to_one_chat.chat_options.click()
+        self.device_1_one_to_one_chat.view_profile_button.click()
+        self.device_1_one_to_one_chat.remove_from_contacts.click_until_absense_of_element(self.device_1_one_to_one_chat.remove_from_contacts)
+        self.device_1_one_to_one_chat.close_button.click()
+
+        self.device_1.just_fyi('Device1 removes 1-1 chat from home screen (for the next test)')
+        self.home_1.home_button.double_click()
```

Comment:
better to do preparation for next test in next test

Reply:
done